src/app/redux/TransactionSaga.js

```diff
@@ -125,15 +126,23 @@ function* error_account_witness_vote({operation: {account, witness, approve}}) {
 
 /** Keys, username, and password are not needed for the initial call.  This will check the login and may trigger an action to prompt for the password / key. */
 function* broadcastOperation({payload:
-    {type, operation, confirm, warning, keys, username, password, successCallback, errorCallback}
+    {type, operation, confirm, warning, keys, username, password, successCallback, errorCallback, postUnsafe}
 }) {
-    const operationParam = {type, operation, keys, username, password, successCallback, errorCallback}
+    const operationParam = {type, operation, keys, username, password, successCallback, errorCallback, postUnsafe}
+
     const conf = typeof confirm === 'function' ? confirm() : confirm
     if(conf) {
         yield put(tr.actions.confirmOperation({confirm, warning, operation: operationParam, errorCallback}))
         return
     }
     const payload = {operations: [[type, operation]], keys, username, successCallback, errorCallback}
+    if (!postUnsafe && hasPrivateKeys(payload)) {
```

Comment:
`!postUnsafe && ...`? if the post is unsafe I would expect postUnsafe to be falsy, rather than truthy. What's happening here?

Reply:
If you read it as `allowPostUnsafe` maybe it makes sense? I'm no fan of how redux works and that's the best way I could figure out how to show that modal, happy to hear an alternative

Reply:
`allowPostUnsafe` is a fantastic name for this purpose.